lib/signaling/v2/peerconnection.js

```diff
@@ -279,8 +280,11 @@ class PeerConnectionV2 extends StateMachine {
    * @returns {Promise<void>}
    */
   _addIceCandidate(candidate) {
+    if (!(candidate || isEdge)) {
```

Comment:
@syerrapragada This can be simplified to:
```js
return Promise.resolve().then(() => {
  if (candidate || isEdge) {
    candidate = candidate && new this._RTCIceCandidate(candidate);
    return this._peerConnection.addIceCandidate(candidate);
  }
});
```